lib/widgets/list_cocktail.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:boilerplate/styles/colors.dart';
+import 'package:boilerplate/styles/size.dart';
+import 'package:boilerplate/styles/spacing.dart';
+import 'package:flutter/material.dart';
+
+class ListCocktail extends StatelessWidget {
+  final List<dynamic> list;
```

Comment:
Using `dynamic` types sometimes can be way too free and be prompt to weird errors and crashes in production. 
Can we use instead of `dynamic` a declared type or a generic `interface` that works for everything?